src/lib/Microsoft.Health.Fhir.Ingest/Config/ResourceIdentityOptions.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.Health.Fhir.Ingest.Config
 {
     public class ResourceIdentityOptions
     {
-        public string ResourceIdentityServiceType { get; set; } = "R4DeviceAndPatientLookupIdentityService";
+        public string ResourceIdentityServiceType { get; set; } = "Lookup";

```

Comment:
I would update this to be your enumeration, i.e. ResourceIdentityServiceType.Lookup.ToString().

Reply:
Updated.